### PR TITLE
fix markdown typos on checklist.md file

### PR DESCRIPTION
**Pull request by SimonLab**
see: https://github.com/dwyl/product-roadmap/pull/13/files#r344675348

### CODE REVIEW

File: checklists.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ https://www.npr.org/templates/story/story.php?storyId=122226184 <br />
 [Shane Parrish](https://fs.blog/2014/02/the-checklist-manifesto),
 [Malcolm Gladwell](http://atulgawande.com/book/the-checklist-manifesto),
 [Vinod Khosla](https://twitter.com/vkhosla/status/222469073806700544),
-[Bill Gates](https://www.gatesnotes.com/Books/Checklist-Manifesto), etc.
```
Comment (SimonLab) [inline checklists.md:59]: Unfortunately the link returns a 404 page and looking for the book doesn't return any matching result:
image

Comment (iteles) [reply checklists.md:59]: https://www.analogue.network/book/the-checklist-manifesto-wp3t87?u=billgates I think has it

Comment (nelsonic) [reply checklists.md:59]: Indeed the existing link appears to **`404`**:
image

@iteles thanks for tracking down an alternative link. 👍

**Review by nelsonic — APPROVED** [review]
@SimonLab thanks for making these updates/improvements. 🎉 
If you have time to update the link as shared by @iteles https://github.com/dwyl/product-roadmap/pull/18/files#r344707331 please do. 
Otherwise good to merge. 👍

File: checklists.md (hunk 1 above)
Comment (SimonLab) [reply checklists.md:59]: done :arrow_down:

**Review by iteles — APPROVED** [review]
🎉 Thanks @SimonLab !

**Pull request merged**